### DISCUSSION TITLE
enabling neopixel menu

### DISCUSSION THREAD

**Discussion by VaBanck** [mriscoc/Ender3V2S1#1271] [General]
after turning on the neopixel menu, it gives the following error
![Снимок экрана 2024-03-11 в 18 15 54](https://github.com/mriscoc/Ender3V2S1/assets/75133860/3c9a9dc7-f2bd-4f2c-ba4d-49922bedd567)
![Снимок экрана 2024-03-11 в 18 11 49](https://github.com/mriscoc/Ender3V2S1/assets/75133860/f4c1c08b-6186-4abb-943c-af75812cae72)

**Comment by heidepiek** [comment]
If you read this thread carefully: https://github.com/mriscoc/Ender3V2S1/issues/1254
And alter a typo in the headerfile.
I think you can compile without errors.

Greetz Harrie

**Reply by VaBanck** [reply]
Thanks

**Reply by VaBanck** [reply]
![image](https://github.com/mriscoc/Ender3V2S1/assets/75133860/54ef2252-25ab-4673-91e1-904034106960)

**Reply by heidepiek** [reply]
Nice that it worked out with the neopixels. Impressive hotend! Never seen it before. You are a real tinkerer too I see. Nice work!
![ender 3v2 blue sprite](https://github.com/mriscoc/Ender3V2S1/assets/104028471/1ff1c738-7190-4955-8b32-cb868d4d14b9)

**Reply by VaBanck** [reply]
https://k3d.tech/ebp/releases/

Russian version
